beacon-chain/sync/initial-sync/service.go

```diff
@@ -178,6 +178,7 @@ func (s *Service) Syncing() bool {
 func (s *Service) Resync() error {
 	// set it to false since we are syncing again
 	s.synced = false
+	// How does the validator handle this?
```

Comment:
Should there be anything special done here? Missed it in the beacon-chain PR.

Reply:
Don't think you can, you would have to create new methods in the validator client to handle this.